CMakeLists.txt

```diff
@@ -445,11 +466,14 @@ target_include_directories(${PROJECT_NAME} PUBLIC $<BUILD_INTERFACE:${CMAKE_CURR
 if (BUILD_TESTING)
     enable_testing()
 
+    ############################################################################
+    ################### build testlib (utility library) ########################
+    ############################################################################
+
     file(GLOB TESTLIB_SRC "tests/testlib/*.c")
     file(GLOB EXAMPLES_SRC "docs/examples/*.c")
-    file(GLOB TESTLIB_HEADERS "tests/testlib/*.h" "tests/s2n_test.h")
 
-    add_library(testss2n STATIC ${TESTLIB_HEADERS} ${TESTLIB_SRC} ${EXAMPLES_SRC})
```

Comment:
Headers shouldn't be included as part of the `add_library` directive. Generally the `add_library` directive is just used for things that generate object files. Header should then be added through `target_include_directories`, but that is already configured to add the `tests/` path.